include/aie/Dialect/AIE/AIENetlistAnalysis.h

```diff
@@ -33,7 +33,7 @@ namespace AIE {
 
 class NetlistAnalysis {
```

Comment:
FWIW, I think this whole thing should go away.

Reply:
next one